### PR TITLE
Fix issue #5029: [In the src/index.tsx file , changed selectsRange type to boolean insted of never & true]

### PR DESCRIPTION
**Pull request by nmpavel**
---
name: Pull Request
about: Create a pull request to improve this repository
title: "Changed selectsRange type to boolean insted of never & true"
labels: ""
assignees: ""
---

## Description
**Linked issue**: #5029

**Problem**
Change type of selectsRange back to a boolean

**Changes**
In the src/index.tsx file , changed selectsRange type to boolean insted of never & true.

## To reviewers
Not a big change to write test , just changed the type and that's it.

## Contribution checklist
- [Yes] I have followed the [contributing guidelines](https://github.com/Hacker0x01/react-datepicker/blob/main/CONTRIBUTING.md).
- [ No] I have added sufficient test coverage for my changes.
- [ Yes] I have formatted my code with Prettier and checked for linting issues with ESLint for code readability.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -4,33 +4,33 @@
 
 "@aashutoshrathi/word-wrap@^1.2.3":
   version "1.2.6"
-  resolved "https://registry.yarnpkg.com/@aashutoshrathi/word-wrap/-/word-wrap-1.2.6.tgz#bd9154aec9983f77b3a034ecaa015c2e4201f6cf"
+  resolved "https://registry.npmjs.org/@aashutoshrathi/word-wrap/-/word-wrap-1.2.6.tgz"
```
[the remark this answers is not in the document]
Comment (nmpavel) [reply yarn.lock:7]: Sorry ,this was my first open source contribution. There was some errors I was not sure which eventually fell into some unwanted changes. I'll solve the issues and request for another pull request.

File: src/index.tsx [hunk 2]
```diff
@@ -204,7 +204,7 @@ export type DatePickerProps = OmitUnion<
     ) => void;
   } & (
     | {
-        selectsRange?: never;
+        selectsRange?: boolean;
```
Comment (laug) [inline src/index.tsx:207]: This should probably be 'false' instead of 'boolean'

File: src/index.tsx [hunk 3]
```diff
@@ -224,7 +224,7 @@ export type DatePickerProps = OmitUnion<
         ) => void;
       }
     | {
-        selectsRange?: never;
+        selectsRange?: boolean;
```
Comment (laug) [inline src/index.tsx:227]: This should probably be `false` instead of `boolean`

File: src/index.tsx [hunk 4]
```diff
@@ -214,7 +214,7 @@ export type DatePickerProps = OmitUnion<
         ) => void;
       }
     | {
-        selectsRange: true;
+        selectsRange: boolean;
```
Comment (laug) [inline src/index.tsx:217]: This should probably remain `true` instead of being changed to `boolean`

**Pull request closed without merging**